hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -1717,18 +1717,18 @@ public static Throwable toException(final NameBytesPair parameter) throws IOExce
    * A helper to retrieve region info given a region name
    * using admin protocol.
    *
-   * @param admin
-   * @param regionName
    * @return the retrieved region info
    * @throws IOException
    */
-  public static org.apache.hadoop.hbase.client.RegionInfo getRegionInfo(final RpcController controller,
-      final AdminService.BlockingInterface admin, final byte[] regionName) throws IOException {
+  public static org.apache.hadoop.hbase.client.RegionInfo getRegionInfo(RpcController controller,
+      AdminService.BlockingInterface admin, byte[] regionName) throws IOException {
     try {
       GetRegionInfoRequest request =
+          org.apache.hadoop.hbase.client.RegionInfo.isEncodedRegionName(regionName)?
```

Comment:
javadoc should make note that we're trying to automatically handle encoded region names.

Reply:
Sure